src/main/scala/tech/cryptonomic/conseil/Lorre.scala

```diff
@@ -35,6 +36,7 @@ object Lorre extends App with TezosErrors with LazyLogging with LorreAppConfig w
   //the dispatcher is visible for all async operations in the following code
   implicit val system: ActorSystem = ActorSystem("lorre-system")
   implicit val dispatcher = system.dispatcher
+  implicit val nodeContext = RemoteContext(tezosConf, callsConf, streamingClientConf)
```

Comment:
RemoteContext is an object under TezosRemote Instances, not a priority right now, but maybe in the future we want this to be a level higher? I do like this call though, because it's clear that tezosConf determines what remote context were accessing.